### PR TITLE
small responsive tweak

### PR DESCRIPTION
**Pull request by pveyes**
#1 

**Before**
Screen Shot 2021-07-05 at 11 16 09

**After**
Screen Shot 2021-07-05 at 11 16 16

### CODE REVIEW

**Comment by tibudiyanto** [comment]
Thank you gan. merginggg

File: src/pages/index.js [hunk 1]
```diff
@@ -50,20 +51,20 @@ const VaxLocation = (location) => {
...
         </Text>
         <Text>{wilayah}</Text>
-        <Stack direction="row">
+        <Stack direction="row" wrap="wrap" gridRowGap={2} paddingBlockEnd={2}>
           {jadwal.map(({ id, waktu }) => {
             return (
               <Popover key={id}>
                 <PopoverTrigger>
-                  <Button>{id}</Button>
+                  <Button margin={0}>{id}</Button>
```
Comment (pveyes) [inline src/pages/index.js]: whoops, seems like margin-left in this button doesn't work, will revert

Comment (tibudiyanto) [reply src/pages/index.js]: Eh ok will hold the merge

Comment (pveyes) [reply src/pages/index.js]: done

**Pull request merged**